### PR TITLE
DAOS-11635 doc: fixes to QSG setup guide

### PR DESCRIPTION
**Pull request by Michael-Hennecke**
fixes to QSG setup guides (rhel and suse)

Doc-only: true
Signed-off-by: Michael Hennecke <michael.hennecke@intel.com>

### CODE REVIEW

File: docs/QSG/setup_rhel.md [hunk 1]
```diff
@@ -310,11 +309,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/relea
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos0
-			scm_class: dcpm
-			scm_list: [/dev/pmem0]
-			bdev_class: nvme
-			bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /tmp/pmem0
```
Comment (kccain) [inline docs/QSG/setup_rhel.md:315]: should this scm_mount value be /mnt/daos0 instead?

File: docs/QSG/setup_rhel.md [hunk 2]
```diff
@@ -325,11 +329,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/relea
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos1
-			scm_class: dcpm
-			scm_list: [/dev/pmem1]
-			bdev_class: nvme
-			bdev_list: ["0000:83:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /tmp/pmem1
```
Comment (kccain) [inline docs/QSG/setup_rhel.md:335]: should this scm_mount value be /mnt/daos1 instead?

File: docs/QSG/setup_suse.md [hunk 3]
```diff
@@ -328,11 +320,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/relea
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos0
-			scm_class: dcpm
-			scm_list: [/dev/pmem0]
-			bdev_class: nvme
-			bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /mnt/pmem0
```
Comment (kccain) [inline docs/QSG/setup_suse.md:326]: should this scm_mount value be /mnt/daos0 instead?

File: docs/QSG/setup_suse.md [hunk 4]
```diff
@@ -343,19 +340,26 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/relea
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos1
-			scm_class: dcpm
-			scm_list: [/dev/pmem1]
-			bdev_class: nvme
-			bdev_list: ["0000:83:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /mnt/pmem1
```
Comment (kccain) [inline docs/QSG/setup_suse.md:346]: should this scm_mount value be /mnt/daos1 instead?

**Review by kccain — CHANGES_REQUESTED** [review]

File: docs/QSG/setup_suse.md (hunk 4 above)
Comment (Michael-Hennecke) [reply docs/QSG/setup_suse.md]: thanks for catching - fixed

File: docs/QSG/setup_suse.md (hunk 3 above)
Comment (Michael-Hennecke) [reply docs/QSG/setup_suse.md]: thanks for catching - fixed

File: docs/QSG/setup_rhel.md (hunk 1 above)
Comment (Michael-Hennecke) [reply docs/QSG/setup_rhel.md]: thanks for catching - fixed

**Review by kccain — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**